editor/src/components/canvas/canvas-utils.ts

```diff
@@ -2507,7 +2508,12 @@ export function cullSpyCollector(
   })
   // Eliminate the scene paths which are invalid.
   fastForEach(Object.keys(spyCollector.current.spyValues.scenes), (scenePath) => {
-    if (!scenePaths.has(scenePath)) {
+    if (
+      !scenePaths.has(scenePath) &&
+      !elementPaths.has(
+        TP.toString(TP.instancePath([], (TP.fromString(scenePath) as ScenePath).sceneElementPath)),
```

Comment:
I _believe_ you can pull the scene path object from `spyCollector.current.spyValues.scenes`, then invoke `sceneElementPath` on that. We _really_ don't want to be running `TP.fromString` if we can help it.